panel/src/components/Views/Users/UserInfo.vue

```diff
@@ -31,7 +31,7 @@ export default {
 	height: var(--height-lg);
 	gap: 0.75rem;
 	padding-inline: var(--spacing-2);
-	background: var(--color-white);
+	background: light-dark(var(--color-white), var(--color-gray-850));
```

Comment:
```suggestion
	background: var(--item-color-back);
```

I think we could reuse this here.

Reply:
Ahh I'm was not aware of this var. Nice.